lib/versioner/external.go

```diff
@@ -51,16 +50,11 @@ func (v External) Archive(filePath string) error {
 
 	l.Debugln("archiving", filePath)
 
-	inFolderPath, err := filepath.Rel(v.folderPath, filePath)
-	if err != nil {
-		return err
-	}
-
 	if v.command == "" {
 		return errors.New("Versioner: command is empty, please enter a valid command")
 	}
 
-	cmd := exec.Command(v.command, v.folderPath, inFolderPath)
+	cmd := exec.Command(v.command, v.filesystem.Type(), v.filesystem.URI(), filePath)
```

Comment:
We should move to templated commands and as part of config migration prepend a template that defaults to the current args we add to the command.